KhanhNT/Openstack/3.Architechture.md

```diff
@@ -0,0 +1,31 @@
+# Kiến trúc Openstack
+- Hệ thống yêu cầu ít nhất 2 Nodes(host) để có thể tạo được các `virtual machine (instance)` cơ bản.
```

Comment:
Sai nhé. 1 node cũng đc miễn đủ tài nguyên để chạy,  có thể cài all in one các service trên cùng 1 host